web/app.js

```diff
@@ -1765,6 +1767,7 @@ const PDFViewerApplication = {
       return; // The document was closed while the metadata resolved.
     }
     this.documentInfo = info;
+    this.rawMetadata = rawMetadata;
```

Comment:
With the suggested changes in `src/display/api.js` and `src/display/metadata.js`, this is no longer needed.